graphql-lwt/test/lwt_test.ml

```diff
@@ -45,6 +50,15 @@ let suite = [
       ]
     ])
   );
+  ("error reporting", `Quick, fun () ->
+    test_query schema () "{ io_error }" (`Assoc [
+      "data", `Assoc [
+        "io_error", `Null
+      ];
+      "errors", `List [
+        `Assoc [ "message", `String "error" ]
+      ]
+    ]))
```

Comment:
I'm not sure this test case adds much beyond what's already covered in [`error_test.ml`](https://github.com/osener/ocaml-graphql-server/blob/dddb241fb679a18605ff66f6ef650d93bb71e7b0/graphql/test/error_test.ml).

It would be great to start a new suite of tests for `Graphql_lwt.Server` though, which is currently untested. I don't consider that a requirement for merging this PR though.

Reply:
Since this bug happened in graphql-lwt, I thought it would be nice to add a test that checks if errors are returned as objects. Thanks for the pointer to the `error_test` module, I missed that it does test this exact behavior. I'll remove the test from the commit.